server/client/src/Routes/PublicRoute.js

```diff
@@ -0,0 +1,11 @@
+import {
+    Navigate
+  } from 'react-router-dom';
+  
+  function PublicRoute({ children, isAuthenticated}) {
```

Comment:
Slightly not  implemented correctly, see this link [example](https://dev.to/nilanth/how-to-create-public-and-private-routes-using-react-router-72m#:~:text=The%20private%20route %20component%20is,is%20authenticated%20(Logged%20in).) This component should return route or redirect